test/release/examples/primers/taskParallel.chpl

```diff
@@ -97,3 +97,72 @@ coforall i in 1..n {
 // The order of output is undefined.
 writeln("5: output from main task");
 
+/*
+.. _primers-taskparallel-task-intents:
+
+Task Intents
+------------
+
+The body of a task construct name some variables declared outside,
+or "outer variables". If so, "shadow variables" are introduced.
```

Comment:
This doesn't seem to parse.  "name some" -> "may refer to variables"?  "outside" -> "outside the task's lexical scope"?  "If so" -> "When it does"